org-wild-notifier.el

```diff
@@ -286,43 +282,46 @@ EVENT-MSG is a string representation of the event."
 	 :title org-wild-notifier-notification-title
 	 :severity org-wild-notifier--alert-severity))
 
-(defun org-wild-notifier--extract-time (event)
-  "Extract timestamps from EVENT.
+(defun org-wild-notifier--extract-time (marker)
+  "Extract timestamps from MARKER.
 Timestamps are extracted as cons cells.  car holds org-formatted
 string, cdr holds time in list-of-integer format."
   (-non-nil
    (--map
-    (let ((org-timestamp (cdr (assoc it event))))
+    (let ((org-timestamp (org-entry-get marker it)))
       (and org-timestamp
            (cons org-timestamp
                  (apply 'encode-time (org-parse-time-string org-timestamp)))))
     '("DEADLINE" "SCHEDULED" "TIMESTAMP"))))
 
-(defun org-wild-notifier--extract-title (event)
-  "Extract event title from EVENT."
-  (cdr (assoc "ITEM" event)))
+(defun org-wild-notifier--extract-title (marker)
+  "Extract event title from MARKER.
+MARKER acts like the event's identifier."
+  (org-with-point-at marker
+    (-let (((_lvl _reduced-lvl _todo _priority title _tags)
+            (org-heading-components)))
+      title)))
 
-(defun org-wild-notifier--extract-notication-intervals (event)
+(defun org-wild-notifier--extract-notication-intervals (marker)
   "Extract notification intervals from the event's properties.
-Resulting list also contains standard notification
-interval (`org-wild-notifier-alert-time')."
+MARKER acts like the event's identifier.  Resulting list also contains
+standard notification interval (`org-wild-notifier-alert-time')."
   `(,@(-flatten (list org-wild-notifier-alert-time))
     ,@(-map 'string-to-number
-            (cdr (assoc org-wild-notifier-alert-times-property event)))))
+           (org-entry-get-multivalued-property
+            marker
+            org-wild-notifier-alert-times-property))))
 
-(defun org-wild-notifier--gather-info (event)
-  "Collect information about an event."
-  `((times . (,(org-wild-notifier--extract-time event)))
-    (title . ,(org-wild-notifier--extract-title event))
-    (intervals . ,(org-wild-notifier--extract-notication-intervals event))
-    (event . ,event)))
+(defun org-wild-notifier--gather-info (marker)
+  "Collect information about an event.
+MARKER acts like event's identifier."
+  `((times . (,(org-wild-notifier--extract-time marker)))
+    (title . ,(org-wild-notifier--extract-title marker))
+    (intervals . ,(org-wild-notifier--extract-notication-intervals marker))))
 
 (defun org-wild-notifier--stop ()
-  "Stops the notification timer and cancel any in-progress checks."
```

Comment:
this doesn't look right

Reply:
Hmm, I don't think I changed this in the original commits.